examples/action/ETH/ETH.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE token  [
+    <!ENTITY style SYSTEM "shared.css">
+    <!ENTITY item-view-minted.en SYSTEM "item-view-supply.en.js">
+    <!ENTITY style SYSTEM "shared.css">
+    <!ENTITY deposit.en SYSTEM "supply.en.js">
+]>
+<ts:token xmlns:ts="http://tokenscript.org/2020/06/tokenscript"
+          xmlns:ethereum="urn:ethereum:constantinople"
+          xmlns:xhtml="http://www.w3.org/1999/xhtml"
+          xmlns:xml="http://www.w3.org/XML/1998/namespace"
+          xmlns:asnx="urn:ietf:params:xml:ns:asnx"
+          xsi:schemaLocation="http://tokenscript.org/2020/06/tokenscript http://tokenscript.org/2020/06/tokenscript.xsd"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          custodian="false"
+>
+    <asnx:module name="aETH">
+        <namedType name="MintOnDeposit">
+            <type>
+                <sequence>
+                    <element name="_fromMinter" ethereum:type="address" ethereum:indexed="true"/>
+                    <element name="_valueMinted" ethereum:type="uint256" ethereum:indexed="false"/>
+                    <element name="_fromBalanceIncreaseMinted" ethereum:type="uint256" ethereum:indexed="false"/>
+                    <element name="_fromIndexMinted" ethereum:type="uint256" ethereum:indexed="false"/>
+                </sequence>
+            </type>
+        </namedType>
+        <namedType name="Transfer">
+            <type>
+                <sequence>
+                    <element name="from" ethereum:type="address" ethereum:indexed="true"/>
+                    <element name="to" ethereum:type="address" ethereum:indexed="true"/>
+                    <element name="amount" ethereum:type="uint256" ethereum:indexed="false"/>
+                </sequence>
+            </type>
+        </namedType>
+    </asnx:module>
+
+    <ts:label>
+        <ts:string xml:lang="en">ETH</ts:string>
+    </ts:label>
+
+    <ts:contract name="balance">
+        <ts:address network="1">0xb8c9fc923e67da6d7979bf2c290bd1df9c512bfa</ts:address>
+    </ts:contract>
+    <ts:contract name="aETH" interface="erc20">
+        <ts:address network="1">0x3a3a65aab0dd2a17e3f1947ba16138cd37d08c04</ts:address>
+    </ts:contract>
+    <ts:contract name="ETH">
```

Comment:
Can you make it specify that it's "erc20", like this? :

```
<ts:contract name="ETH" interface="erc20">
```

The only thing effects this has currently is to add the default actions for that `interface`. In this case the ERC20 send and receive actions, which behave the same way for native crypto in the app.

The "Deposit" action seems to show up and gets displayed

Reply:
@hboon it won't confuse that for an erc20 contract in terms of logic tho? I guess you will catch that in the code base

Reply:
Not at the moment. Setting it is only makes the send and receive button appear in that token screen shared by fungibles. When those 2 buttons are tapped, it will sort out of it’s an ERC20 or native crypto. 

Right after the train, we might to fix it though since this would signify inheritance of the base TokenScript.